src/registry/routes/helpers/get-component-fallback.js

```diff
@@ -4,28 +4,63 @@ var request = require('minimal-request');
 var url = require('url');
 var _ = require('underscore');
 
-module.exports = function(fallbackRegistryUrl, headers, component, callback) {
-  return request({
-    method: 'post',
-    url: fallbackRegistryUrl,
-    headers: _.extend({}, headers, {'host': url.parse(fallbackRegistryUrl).host}),
-    json: true,
-    body: {
-      components: [
-        component
-      ]
+module.exports = {
+  getComponent: function (fallbackRegistryUrl, headers, component, callback) {
+    return request({
+      method: 'post',
+      url: fallbackRegistryUrl,
+      headers: _.extend({}, headers, {'host': url.parse(fallbackRegistryUrl).host}),
+      json: true,
+      body: {
+        components: [
+          component
+        ]
+      }
+    }, function (err, res) {
+      if (err || !res || res.length === 0) {
+        return callback({
+          status: 404,
+          response: {
+            code: 'NOT_FOUND',
+            error: err
+          }
+        });
+      }
+
+      return callback(res[0]);
+    });
+  },
+  getComponentInfoOrPreviewFallback: function (conf, req, res, localRegistryError, component, callback) {
```

Comment:
I think I prefer if we keep it modularised to keep consistency, example `get-component-info-fallback.js` and `get-component-fallback.js`. Each thing exposes one thing with the same name.